### PR TITLE
Added repo version command - Issue #14

### PR DESCRIPTION
**Pull request by asi7296**
Added a new command to display 7-character hash of the latest commit, pulled from the response text from a GET request to the master repo Github page.

### CODE REVIEW

**Comment by joaorafaelm** [comment]
@asi7296 there is an env var from heroku that contains the commit that is in production (HEROKU_SLUG_COMMIT) https://devcenter.heroku.com/articles/dyno-metadata so there is no need for requests lib here.
I would also call the command '/version' to keep it simple.

It would also be nice if there was a test for this command, but if you dont have the time I can do it later :)

**Comment by asi7296** [comment]
@joaorafaelm sounds good, I'll make the changes. The heroku environment variables can be accessed through the os.environ module when deployed on that dyno, correct?

**Comment by asi7296** [comment]
@joaorafaelm I've made the changes, I'm not entirely sure how to write those tests though.

File: markov.py [hunk 1]
```diff
@@ -60,6 +61,16 @@ def remove_messages(message):
     bot.reply_to(message, 'u r not an admin 🤔')
 
 
+@bot.message_handler(commands=['version'])
+def get_repo_version(message):
+	hash_len = 7
```
Comment (joaorafaelm) [inline markov.py]: could you change from tabs to spaces please? 4 spaces :)

File: markov.py [hunk 2]
```diff
@@ -60,6 +61,16 @@ def remove_messages(message):
...
+	try:
```
Comment (joaorafaelm) [inline markov.py]: the try block is not necessary. 
I think just the bot.reply_to(message, environ.get('HEROKU_SLUG_COMMIT')[:hash_len]) is fine.
in fact you could define the method like this:
```
hash_len = 7
commit_hash = environ.get('HEROKU_SLUG_COMMIT')[:hash_len]
bot.reply_to(message, commit_hash)
```

**Comment by joaorafaelm** [comment]
The test can be written like this: https://github.com/joaorafaelm/markov-bot/blob/master/test_markov.py#L40

Example:
```python
@mock.patch('markov.bot')
def test_generate_sentence(mock_bot, message):
    markov.get_repo_version(message)
    assert mock_bot.reply_to. mock_model.called_once_with(message)
```

File: markov.py (hunk 2 above)
Comment (joaorafaelm) [reply markov.py]: the .get method should also return a empty string as default, in case there is no env var called HEROKU_SLUG_COMMIT

Comment (joaorafaelm) [reply markov.py]: like `environ.get('HEROKU_SLUG_COMMIT', '')[:hash_len]`

File: markov.py (hunk 1 above)
Comment (asi7296) [reply markov.py]: Done :)

File: markov.py [hunk 3]
```diff
@@ -4,6 +4,7 @@
 import dataset
 from cachetools.func import ttl_cache
 import logging
+import requests
```
Comment (joaorafaelm) [inline markov.py]: you can remove this import

**Comment by asi7296** [comment]
@joaorafaelm I've added the tests as well.

**Pull request merged**